docs/getting-started/keeping-backstage-updated.md

```diff
@@ -0,0 +1,33 @@
+---
+id: keeping-backstage-updated
+title: Keeping Backstage Updated
+description: How to keep your Backstage App updated
+---
+
+Backstage is always improving, so it's a good idea to stay in sync with the
+latest releases. Backstage is more of a library than an application or service;
+similar to `create-react-app`, the `@backstage/create-app` tool gives you a
+starting point that's meant to be evolved.
+
+## Updating Backstage versions with backstage-cli
+
+The Backstage CLI has a command to bump all `@backstage` packages you're using
+to the latest versions:
+[versions:bump](https://backstage.io/docs/cli/commands#versionsbump).
```

Comment:
if i'm not mistaken, it will actually bump ANYTHING, won't it?  i believe i've run it before and it's offered that I should bump my `@roadiehq/backstage-plugin-xxxxx` plugins, as well?  it's almost more dependabot-like, ignorant to WHO the author is, so wonder if this should avoid saying "bump all `@backstage`..."

Reply:
Hmm, the logic in [bump](https://github.com/backstage/backstage/blob/master/packages/cli/src/commands/versions/bump.ts) strictly looks for `@backstage`-prefixed packages initially, but there's some additional logic there that might try to bump related packages with dependencies on `@backstage` 🤔

Reply:
i'll defer on that one then 😁 the intent of improving is definitely made

Reply:
I changed it to `@backstage packages and dependencies` at least 😄